### PR TITLE
Don't initialize supervisor object for ingress

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

The ingress panel only used the supervisor object for translations.
By not initializing the object ingress panels will load faster.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: hassio/src/ingress-view/hassio-ingress-view.ts [hunk 1]
```diff
@@ -102,14 +99,14 @@ class HassioIngressView extends LitElement {
         }
         if (!addonInfo.version) {
           await showAlertDialog(this, {
-            text: this.supervisor.localize("my.error_addon_not_installed"),
+            text: this.hass.localize("supervisor.my.error_addon_not_installed"),
```
Comment (bramkragten) [inline hassio/src/ingress-view/hassio-ingress-view.ts:102]: Didn't we strip the supervisor translations from the frontend translations?

Comment (ludeeus) [reply hassio/src/ingress-view/hassio-ingress-view.ts:102]: Yes, looks like this PR is missing the filechange to change that.
And the supervisor localize is still initialized, it should not be

File: build-scripts/gulp/translations.js [hunk 2]
```diff
@@ -266,7 +266,6 @@ gulp.task(taskName, function () {
         TRANSLATION_FRAGMENTS.forEach((fragment) => {
           delete data.ui.panel[fragment];
         });
-        delete data.supervisor;
```
Comment (bramkragten) [inline build-scripts/gulp/translations.js:269]: So we are now shipping all supervisor localizations to the frontend, always, just for the few ingress error messages? Doesn't it make more sense to make a separate section for those and duplicate them with lokalise keys?

File: hassio/src/ingress-view/hassio-ingress-view.ts [hunk 3]
```diff
@@ -102,14 +99,14 @@ class HassioIngressView extends LitElement {
...
             title: addonInfo.name,
           });
           await nextRender();
           navigate(`/hassio/addon/${addonInfo.slug}/info`, { replace: true });
         } else if (!addonInfo.ingress) {
           await showAlertDialog(this, {
-            text: this.supervisor.localize("my.error_addon_no_ingress"),
+            text: this.hass.localize("supervisor.my.error_addon_no_ingress"),
```
Comment (bramkragten) [inline hassio/src/ingress-view/hassio-ingress-view.ts:109]: Add a fallback for the translations, as we now use the HA translations, older HA versions will not have them.

**Pull request closed without merging**